library/Ivoz/Provider/Domain/Model/Carrier/Carrier.php

```diff
@@ -36,6 +36,14 @@ public function getId(): ?int
      */
     protected function sanitizeValues(): void
     {
+        $billingEnabled = $this
+            ->getBrand()
+            ->hasFeatureByIden('billing');
```

Comment:
We should add constants for these someday. They exist for ids which seems much worse than hardcoding idens.

Reply:
I agree. Added to backlog